client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```
@@ -249,19 +246,100 @@ h2.action-panel__title {
 	color: var( --color-neutral-70 );
 }
 
-.email-provider-stacked-card__header {
-	display: flex;
-	flex-direction: row;
-	flex-wrap: wrap;
-	justify-content: space-between;
+.email-providers-stacked-comparison__provider-card {
+	.email-provider-stacked-card__header {
+		display: flex;
+		flex-direction: column;
+		flex-wrap: wrap;
+
+		.email-provider-stacked-card__provider-card-main-details {
+			margin-top: 12px;
+			width: calc( 100% - 10px );
+			margin-left: 10px;
+			@include break-large {
+				margin-top: 0px;
+				margin-left: 10px;
+				width: 100px;
+			}
+
+			.email-provider-stacked-card__provider-expand-cta {
+				width: 100%;
+			}
+		}
+
+		@include break-large {
+			flex-direction: row;
+		}
+
+		.email-provider-stacked-card__title-price-badge {
+			width: 100%;
+			margin-left: 10px;
+			flex-basis: content;
+		}
+	}
+	&.is-expanded {
+		.email-provider-stacked-card__header {
+			flex-direction: row;
+			.email-provider-stacked-card__title-price-badge {
+				width: initial;
+				margin-left: 10px;
+			}
+		}
+	}
 }
 
 .email-provider-stacked-card__provider-card-main-details {
 	width: 120px;
 }
 
 .email-provider-stacked-card__title-container {
-	width: $width-left-panel-card;
+	width: 100%;
 	display:flex;
 	flex-direction: column;
+
+	@include break-small {
+		min-width: $width-left-panel-card;
+		flex-basis: content;
+	}
 }
+
+.form-text-input-with-affixes__suffix {
+	font-weight: normal;
+}
+
+.email-providers-stacked-comparison__main.main {
+	margin: 12px auto;
+	@include break-mobile {
+		flex-direction: row;
+	}
+
+	.card.action-panel.promo-card.email-providers-stacked-comparison__provider-card {
+		display: flex;
+		flex-direction: row;
+		.action-panel__figure {
+			margin-top: 3px;
+
+			@include break-mobile {
+				max-width: 24px;
+				order: 1;
+				&.align-left {
+					order: -1;
+				}
```

Nit: can you add whitespace between sections?
```suggestion
.email-providers-stacked-comparison__main.main {
	margin: 12px auto;

	@include break-mobile {
		flex-direction: row;
	}

	.card.action-panel.promo-card.email-providers-stacked-comparison__provider-card {
		display: flex;
		flex-direction: row;

		.action-panel__figure {
			margin-top: 3px;

			@include break-mobile {
				max-width: 24px;
				order: 1;

				&.align-left {
					order: -1;
				}
```